examples/computer/compose.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  default:
+    image: epatey/inspect-computer-tool
+    # image: epatey/inspect-computer-tool:local
+    # x-local: true
+    init: true
+    volumes:
+      - ./flag.txt:/home/computeruse/flag.txt:ro
```

Comment:
Note that samples can have files that are copied into the container (that might be a better way to provide the flag)

Reply:
I've made the change, but let's review to ensure I've done it properly.